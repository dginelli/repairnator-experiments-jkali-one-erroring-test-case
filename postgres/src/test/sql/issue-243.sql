-- should display an ERROR and not also crash the connected ES node
select * from so_posts where zdb('so_posts', ctid) ==> '"';