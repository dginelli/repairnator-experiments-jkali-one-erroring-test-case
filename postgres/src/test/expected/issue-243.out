-- should display an ERROR and not also crash the connected ES node
select * from so_posts where zdb('so_posts', ctid) ==> '"';
ERROR:  rc=500; {"error":{"root_cause":[{"type":"runtime_exception","reason":"#field_lists(title_and_tags=[title,tags], field2=[a,b,c,d], field3=[a,d,q,z], field4=[one,two,three]) (\")"}],"type":"runtime_exception","reason":"#field_lists(title_and_tags=[title,tags], field2=[a,b,c,d], field3=[a,d,q,z], field4=[one,two,three]) (\")","caused_by":{"type":"token_mgr_error","reason":"Lexical error at line 1, column 106.  Encountered: <EOF> after : \"\\\")\""}},"status":500}
